pages/form/edit/[formName].js

```diff
@@ -483,6 +491,42 @@ const FormEditor = () => {
                         },
                     ]}
                 />
+                <Card
+                    width={['500px', '700px', '800px']}
+                    marginBottom={`${theme.space[7]}px`}
+                    marginRight='auto'
+                    marginLeft='auto'
+                    display='flex'
+                    justifyContent='space-between'
+                >
+                    <div>
+                        <TextField
+                            id='filled-basic'
```

Comment:
I don't think this id is serving any purpose right now, right ?

I think you can delete it.